tools/common/Assembly.cs

```diff
@@ -328,6 +328,15 @@ public static LinkWithAttribute GetLinkWithAttribute (CustomAttribute attr)
 			return linkWith;
 		}
 
+		void AddFramework (string file)
+		{
+			Framework framework;
```

Comment:
minor: there's a nicer syntax for this now :)

`TryGetValue (file, out var framework)`